### PR TITLE
Add manifest url path

### PR DESCRIPTION
**Pull request by leonVrashitov**
Add manifest url path
Add GETH keys

### CODE REVIEW

**Review by ConanMishler — COMMENTED** [review]
Has this PR been tested? It looks to me like this line:
```
xbridge_text = autoconfig.load_template(customlist[0]['custom_manifest']+'{}.base.j2'.format(name.lower()))
```
will need to change to this:
```
xbridge_text = autoconfig.load_template(customlist[0]['custom_manifest']+'/autobuild/configs/'+'{}.base.j2'.format(name.lower()))
```
Also, it looks to me like this line:
```
template_wc = Template(autoconfig.load_template(autoconfig.wallet_config())).render(daemon)
```
is preparing to write individual wallet config files based on this generic wallet config file:
```
/autobuild/templates/wallet.conf.j2
```
I think the individual wallet configs should be based instead on the files found here:
```
(default or custom URL)+"/autobuild/configs/{}.base.j2".format(s.lower())
```

**Comment by leonVrashitov** [comment]
Yes, It was tested with autobuild/configs path,  accidentally was removed to test another approach. Forgot to return. WIll fix it.
About /autobuild/templates/wallet.conf.j2. Good point. I assumed these files are constant. But yes, will add individual path to the file.

File: autobuild/app.py [hunk 1]
```diff
@@ -209,8 +209,10 @@ def processconfigs(datalist):
             name = daemon['name']
             if name.upper() not in ['TNODE', 'SNODE', 'TESTSNODE', 'ETH', 'XR_PROXY']:
                 XBRIDGE_CONF += "{},".format(name)
-                template_wc = Template(autoconfig.load_template(autoconfig.wallet_config())).render(daemon)
-
+                if datalist[0]['custom_manifest']:
+                    template_wc = Template(autoconfig.load_template(datalist[0]['custom_manifest']+'autobuild/templates/wallet.conf.j2')).render(daemon)
+                else:
+                    template_wc = Template(autoconfig.load_template(autoconfig.wallet_config())).render(daemon)
```
Comment (ConanMishler) [inline autobuild/app.py:215]: Sorry, my mistake asking you to make this change. Please undo this change.
